vitrina/orgs/models.py

```diff
@@ -76,7 +76,7 @@ class Organization(MP_Node):
     # Deprecated fields
     imageuuid = models.CharField(max_length=36, blank=True, null=True)
 
-    node_order_by = ["created"]
+    node_order_by = ["title"]
```

Comment:
This does not seem to correlate with https://github.com/atviriduomenys/katalogas/blob/252-statistika-ir-filtras-pagal-valdymo-sritis/vitrina/orgs/migrations/0003_auto_20220829_0841.py

It looks, that organization migration does not work.

Reply:
The migration was fixed already in another task.
https://github.com/atviriduomenys/katalogas/pull/234/files#diff-5f95eb53ab5806172ba8d7d60642d1d7dd8107d08e4c83730568165964c5a105